sdks/typescript/client-typescript-eventsource/README.md

```diff
@@ -67,77 +96,90 @@ featureHubRepository.addReadynessListener((readyness) => {
        })
   }
 });
+````
 
-//paste your Service Account SDK URL here
-this.eventSource = new FeatureHubEventSourceClient('<sdkUrl>'); 
-this.eventSource.init();
+#### New Feature Available 
 
-featureHubRepository.getFeatureState('FEATURE_X').addListener((fs: FeatureStateHolder) => {
-  // do something
-});
+If you want to know when a new feature is available, any new feature, then this is the event for you. In particular,
+this only happens after the first events are loaded. It has a caveat that it is triggered when features 
+_actually change_ not when we receive new updates for them. That is particularly important because of the _catch and
+release_ functionality listed below.
+
+```typescript
+featureHubRepository
+  .addPostLoadNewFeatureStateAvailableListener((_) => {
+    // e.g. tell user to page is going to update and re-render page
+});  
 ```
 
+## Updating your repository
```

Comment:
I still struggle to understand how this differs from Imperative requests and Event-driven feature updates described above. I kind of trying to connect the dots and not sure I understand how.